pkgs/top-level/all-packages.nix

```diff
@@ -3716,6 +3720,12 @@ in
 
   sstp = callPackage ../tools/networking/sstp {};
 
+  subsurface =
+    let callpkg = qt55.callPackage;
```

Comment:
Sorry, I should have been more clear. I meant do `qt55.callPackage ./applications/misc/subsurface` rather then `let callpkg = qt55.callPackage in callpkg ./applications/misc/subsurface`.